liota/core/package_manager.py

```diff
@@ -509,23 +534,33 @@ def _package_module_load(self, file_name, path_file_ext, file_ext):
     # This method is called to load package into current Liota process using
     # file_name (no_ext) as package identifier.
 
-    def _package_load(self, file_name, ext_forced=None, check_stack=None):
+    def _package_load(self, file_name, checksum=None, ext_forced=None, check_stack=None):
```

Comment:
The dependency package should be loaded but we cannot expect users to list all those dependency packages' checksums. One way I thought out is to simply refuse that kind of loading request to force users to load packages in order. However, we will lose our flexibility. Therefor, I choose not to decline. If this will be new issue, will use another pull request to fix it. Meanwhile, will talk with security team.